tasks.py

```diff
@@ -536,16 +540,50 @@ def load_datasets(
             cmd = f"poetry run mipdb add-dataset {csv} -d {data_model_code} -v {data_model_version} --copy_from_file {not use_sockets} {get_monetdb_configs_in_mipdb_format(port)} {get_sqlite_path(worker_id)}"
             run(c, cmd)
 
-    # Retrieve and filter worker configurations
+    def load_test_datasets(
```

Comment:
If you load the data into one worker, how does it distinguish between what data should be loaded to the server and the client? I guess it loads both to client and server?